README.md

```diff
@@ -141,6 +142,29 @@ exclude:
 >rake preview:all
 >```
 
+## Docker (docker-compose)
+
+Docker provides a quick and easy way to build and preview the documentation. You do not need to install Ruby dependencies manually. This Docker solution launches Jekyll inside the container in `watch` mode.```
+ruby dependency manually. The provided setup starts Jekyll inside the container in watch mode.
+Every change of a markdown file will automatically trigger a rebuild of the documentation.
+
+1. Install [Docker](https://www.docker.com/get-started). If you have Docker installed, make sure Docker is running.
+
+1. Build and start the Docker environment.
+
+```bash
+docker-compose up
+```
+
+1. Use the server address URL provided by the system response.
+   For example:
+   
+   ```terminal
+   jekyll_1  |     Server address: http://127.0.0.1:4000/
+   jekyll_1  |   Server running... press ctrl-c to stop.
+
```

Comment:
Missing the closing code block. I'm sure that was a GitHub thing too.

Reply:
Yes, already fixed :-)